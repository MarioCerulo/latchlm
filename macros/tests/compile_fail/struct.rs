#![allow(unused)]

use latchlm_core::AiModel;
use latchlm_macros::AiModel;

#[derive(AiModel)]
struct Model;

fn main() {}
