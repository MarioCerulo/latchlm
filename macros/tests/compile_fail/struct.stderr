error: AiModel can only be derived for enums
 --> tests/compile_fail/struct.rs:7:1
  |
7 | struct Model;
  | ^^^^^^^^^^^^^
